03_launch_mgmt_cluster.sh

```diff
@@ -194,29 +234,29 @@ kind: Kustomization
 resources:
 - $(realpath --relative-to="$overlay_path" "$provider_cmpt")
 EOF
-
 }
 
-
 #
 # Launch the cluster-api provider.
 #
 function launch_cluster_api_provider_metal3() {
     pushd "${CAPM3PATH}"
     kustomize_overlay_path=$(mktemp -d capm3-XXXXXXXXXX)
-
-    ./examples/generate.sh -f
-    kustomize_overlay_capm3 "$kustomize_overlay_path" \
-      "$CAPM3PATH/examples/provider-components"
-    kubectl apply -f ./examples/_out/cert-manager.yaml
-    kubectl wait --for=condition=Available --timeout=300s -n cert-manager deployment cert-manager
-    kubectl wait --for=condition=Available --timeout=300s -n cert-manager deployment cert-manager-cainjector
-    kubectl wait --for=condition=Available --timeout=300s -n cert-manager deployment cert-manager-webhook
+    mkdir -p "${HOME}"/.cluster-api
+    touch "${HOME}"/.cluster-api/clusterctl.yaml
+    pushd "${CAPIPATH}"
+    if ! [ -x "$(command -v clusterctl)" ]; then
+      make clusterctl
+    fi
+    popd
 
     pushd "$kustomize_overlay_path"
     update_images
     popd
-    kustomize build "$kustomize_overlay_path" | kubectl apply -f-
+
+    pushd "${CAPIPATH}"
+    cd bin
+    ./clusterctl init --infrastructure=metal3
```

Comment:
if you do it this way, it will download the last release of metal3. We want to acutally test the branch we are on (can be master for daily jobs) so we need to use the override feature.

Reply:
Fixed